src/clj/witan/send/send.clj

```diff
@@ -322,16 +338,17 @@
                           (states/calculate-valid-states-from-setting-academic-years))
 
         valid-year-settings (->> (ds/row-maps valid-setting-academic-years)
-                                 (states/calculate-valid-year-settings-from-setting-academic-years))
-        splice-ncy 11]
+                                 (states/calculate-valid-year-settings-from-setting-academic-years))]
     (s/validate (sc/SENDPopulation+ valid-settings) initial-send-population)
     (s/validate (sc/TransitionsMap+ valid-needs valid-settings) transitions)
     (s/validate (sc/NeedSettingCost+ valid-needs valid-settings) setting-cost)
     {:standard-projection (prep-inputs initial-state splice-ncy valid-states transition-matrix transition-matrix-filtered
-                                       population valid-setting-academic-years original-transitions setting-cost)
+                                       population valid-setting-academic-years original-transitions setting-cost
+                                       filter-transitions-from)
      :scenario-projection (if ((complement nil?) modified-transition-matrix)
-                            (prep-inputs initial-state splice-ncy valid-states modified-transition-matrix transition-matrix-filtered
-                                         population valid-setting-academic-years original-transitions setting-cost)
+                            (prep-inputs initial-state splice-ncy valid-states modified-transition-matrix
```

Comment:
this is a crazy number of arguments, consider using a map for them

Reply:
I feel like a map will add more code but not make anything clearer really. Lots of args here is a bit of a legacy thing, it could do with a whole makeover!

Reply:
ok